packages/fxa-content-server/tailwind.config.js

```diff
@@ -22,8 +22,10 @@ config.theme.extend = {
     'check-white': 'inline("../images/icon-check-white.svg")',
     'show-password': 'inline("../images/icon-show-password.svg")',
     'hide-password': 'inline("../images/icon-show-password-closed.svg")',
-    /* TODO: move this to `fxa-react` , FXA-5745 */
-    'ff-logo': 'url("../images/firefox-logo.svg")',
+    /* TODO: move this to `fxa-react`, FXA-5745 */
+    /* If adding a background-image in content-server, you must refer to "/images" instead of
+     * "../images" because our cache bust build step won't find and replace it in `usemin:css` */
+    'ff-logo': 'url("/images/firefox-logo.svg")',
```

Comment:
Not sure how reliable this sloppy grep is, but this _might_ currently be the only relative `url()` in fxa-content-server:

```sh
git grep -In 'url(' packages/fxa-content-server | grep "\.\./" # on current 'main' branch, not this PR.
# packages/fxa-content-server/tailwind.config.js:26:    'ff-logo': 'url("../images/firefox-logo.svg")',
```

Reply:
Yeah, I think this is the first time we're setting a background-image with a Tailwind class, all the others are inlined (there's a note about that in our `fxa-settings` README btw).